saw-script.cabal

```
@@ -258,11 +268,16 @@ executable saw
     , QuickCheck
     , text
     , transformers
+
+      -- packages in git submodules
+    , aig
+    , cryptol
+
+      -- internal subpackages in the saw tree
+    , cryptol-saw-core
     , saw-script
     , saw-core
     , heapster-saw
```

Similarly, alphabetize these:

```suggestion
    , cryptol-saw-core
    , heapster-saw
    , saw-core
    , saw-script
```

---

Fixed